exampleSite/utilities/examples/content/events/yyyy-city/conduct.md

```diff
@@ -1,6 +1,7 @@
 +++
 Title = "Conduct"
 Type = "event"
+Description = ""
```

Comment:
It's actually way more important than you might think, because otherwise when people share links on Slack, Facebook, etc, it looks like shit. Keep it in there to gently encourage people to think it's a thing they can add, is my opinion. I was going to suggest we could put in some default content, but it would be page-specific (not really that big of a deal if we can do some substituting in the city and year like you do with the data files).

i.e., the `program.md` file could have a default value of:
```
Description = "The program of events for devopsdays $CITY $YEAR"
```
and you replace those values?